.gitignore

```diff
@@ -8,3 +8,4 @@ vendor/
 /phpunit.xml
 /.phpcs-cache
 /phpstan.neon
+/.idea
```

Comment:
Please move this to [your own `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Sorry